src/components/MapBox.tsx

```diff
@@ -96,22 +150,24 @@ export const MapBox: React.FC<{
         />
         <ZoomControl position="bottomleft" />
         {facilities && facilities.length > 0
```

Comment:
you could consider putting these check in a variable, something like 'const hasFacilities = facilities && facilities.length > 0` to make it more readable.

Reply:
Good idea! I didn't think to refactor the existing code. Thanks